README.md

```diff
@@ -20,3 +20,5 @@ $ cd openchpl-sql
 Rename `dev/openchpl_role-template.sql` to `dev/openchpl_role.sql` and set the password for the `openchpl` role. These instructions assume the role/username used for the openchpl database is `openchpl`, and that the password in `openchpl_role.sql`, currently recorded as "change this password" will be update to match your installation. If the installer chooses to change the username/role, make sure it's also changed in the `openchpl.sql` file wherever the role is used.
 
 Next, run the script `dev/reset.sh` or `dev/reset.bat` from the `/dev` directory. These two scripts remove any previous OpenCHPL data model installation, with the associated roles, then recreate the required roles and databases, as well as fill out some of those database schemas with some required information.
+
+Next, follow the procedure.md file to load the CHPL with data. 
```

Comment:
In the previous paragraph, the relative links to the `dev/*` scripts are marked as inline-code and have the directory there. Can you add that here?